### PR TITLE
 Added code of daily Coding Exercise-19

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: First/dailyExercises/JoinArrayList.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package firstPackage.First.dailyExercises;
+
+import java.util.ArrayList;
+
+public class JoinArrayList {
+    public static void main(String args[])
```
Comment (AmrutaDhage20) [inline First/dailyExercises/JoinArrayList.java]: Give space between class and method.

**Review by AmrutaDhage20 — CHANGES_REQUESTED** [review]

File: First/dailyExercises/JoinArrayList.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        // Get the ArrayList1
+        ArrayList<String> listData = new ArrayList<String>();
```
Comment (amitsahu8) [inline First/dailyExercises/JoinArrayList.java]: this is not medium complexity problem

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: First/dailyExercises/JoinArrayList.java (hunk 2 above)
Comment (anushreeSonak) [reply First/dailyExercises/JoinArrayList.java]: Done ! Updated the code

File: First/dailyExercises/BubbleSort.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package firstPackage.First.dailyExercises;
+
```
Comment (AmrutaDhage20) [inline First/dailyExercises/BubbleSort.java]: Give proper package name.

File: First/dailyExercises/BubbleSort.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+public class BubbleSort {
+    public static void bubbleSort(int[] array) {
```
Comment (AmrutaDhage20) [inline First/dailyExercises/BubbleSort.java]: Add space between class and method as per guideline.

**Review by AmrutaDhage20 — CHANGES_REQUESTED** [review]

File: First/dailyExercises/BubbleSort.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+            System.out.print(num + " ");
+        }
+        System.out.println();
+        bubbleSort(array);
+        System.out.println("Array after sorting:");
+        for (int num : array) {
+            System.out.print(num + " ");
+        }
+    }
+}
```
Comment (AmrutaDhage20) [inline First/dailyExercises/BubbleSort.java]: Add output in Readme file.

**Review by AmrutaDhage20 — CHANGES_REQUESTED** [review]

File: First/dailyExercises/BubbleSort.java (hunk 4 above)
Comment (anushreeSonak) [reply First/dailyExercises/BubbleSort.java]: Done

File: First/dailyExercises/BubbleSort.java (hunk 3 above)
Comment (anushreeSonak) [reply First/dailyExercises/BubbleSort.java]: done

File: First/dailyExercises/BubbleSort.java (hunk 5 above)
Comment (anushreeSonak) [reply First/dailyExercises/BubbleSort.java]: Done

File: First/mediumComplexity/READme_BubbleSort.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Write a java program for bubble sort:
+
+
+
```
Comment (AmrutaDhage20) [inline First/mediumComplexity/READme_BubbleSort.md]: Remove extra spaces.

File: First/mediumComplexity/BubbleSort.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
+package firstPackage.First.mediumComplexity;
+
+public class BubbleSort {
+
+    public static void bubbleSort(int[] array) {
+        int data = array.length;
+        for (int index = 0; index < data - 1; index++) {
+            for (int subIndex = 0; subIndex < data - index - 1; subIndex++) {
+                if (array[subIndex] > array[subIndex + 1]) {
```
Comment (amitsahu8) [inline First/mediumComplexity/BubbleSort.java]: not medium complexity

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: First/mediumComplexity/BubbleSort.java (hunk 7 above)
Comment (anushreeSonak) [reply First/mediumComplexity/BubbleSort.java]: Done ! Changed the code

File: First/mediumComplexity/PrintBoundaryElements.java [hunk 8]
```diff
@@ -0,0 +1,41 @@
+package firstPackage.First.mediumComplexity;
+
+public class PrintBoundaryElements {
+    public void Boundary_Elements(int mat[][]){
```
Comment (amitsahu8) [inline First/mediumComplexity/PrintBoundaryElements.java]: Follow the guideline for the method name, variable name

File: First/mediumComplexity/PrintBoundaryElements.java [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+        System.out.println();
+    }
+}
+    public static void main(String[] args)
+    {
+        // Input Matrix
+        int mat[][] = new int[][] { { 1, 2, 3 },
+                { 4, 5, 6 },
+                { 7, 8, 9 } };
+        PrintBoundaryElements B_Values = new PrintBoundaryElements();
```
Comment (amitsahu8) [inline First/mediumComplexity/PrintBoundaryElements.java]: same here

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: First/mediumComplexity/PrintBoundaryElements.java (hunk 9 above)
Comment (anushreeSonak) [reply First/mediumComplexity/PrintBoundaryElements.java]: DONE !

File: First/mediumComplexity/PrintBoundaryElements.java (hunk 8 above)
Comment (anushreeSonak) [reply First/mediumComplexity/PrintBoundaryElements.java]: DONE !

**Review by AmrutaDhage20 — APPROVED** [review]

File: First/mediumComplexity/Reverse.java [hunk 10]
```diff
@@ -0,0 +1,2 @@
+package firstPackage.First.mediumComplexity;public class Reverse {
```
Comment (prashantkawa) [inline First/mediumComplexity/Reverse.java]: remove this file if not needed

File: First/mediumComplexity/PrintBoundaryElements.java [hunk 11]
```diff
@@ -0,0 +1,35 @@
+package firstPackage.First.mediumComplexity;
+
+public class PrintBoundaryElements {
+    public void boundaryElement(int data[][]) {
```
Comment (prashantkawa) [inline First/mediumComplexity/PrintBoundaryElements.java:5]: add space between class and method

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: First/mediumComplexity/PrintBoundaryElements.java (hunk 11 above)
Comment (anushreeSonak) [reply First/mediumComplexity/PrintBoundaryElements.java:5]: Done !

File: First/mediumComplexity/Reverse.java (hunk 10 above)
Comment (anushreeSonak) [reply First/mediumComplexity/Reverse.java]: Done !

**Review by prashantkawa — APPROVED** [review]

**Pull request merged**